src/tools/compiletest/src/runtest.rs

```diff
@@ -1765,6 +1773,10 @@ impl<'test> TestCx<'test> {
         }
 
         rustc.args(&self.props.compile_flags);
+        rustc.arg(&format!(
```

Comment:
...and here. Maybe we want some helper method to make that nicer.

Reply:
I just made this behave as if the user passed `compile-flags: --edition=2018` instead.